# HG changeset patch
# User HackaLittleBit
# Date 1388183244 0
#      Fri Dec 27 22:27:24 2013 +0000
# Node ID 7be8c87533598f5e536cb2b80fb16f45b8e3672f
# Parent  7b8910ca19b87e7af115c3c337f30a8f9fcb8312
shift intercontinental airport 2px to align plane with runways.

diff --git a/src/table/airport_movement.h b/src/table/airport_movement.h
--- a/src/table/airport_movement.h
+++ b/src/table/airport_movement.h
@@ -252,88 +252,88 @@
 
 /** Intercontinental Airport - 4 runways, 8 terminals, 2 dedicated helipads. */
 static const AirportMovingData _airport_moving_data_intercontinental[81] = {
-	AMD(    8,   87, AMED_EXACTPOS,                   DIR_SE), // 00 In Hangar 1
-	AMD(  136,   72, AMED_EXACTPOS,                   DIR_SE), // 01 In Hangar 2
-	AMD(    8,  104, 0,                               DIR_N ), // 02 Taxi to right outside depot 1
-	AMD(  136,   88, 0,                               DIR_N ), // 03 Taxi to right outside depot 2
-	AMD(   56,  120, AMED_EXACTPOS,                   DIR_W ), // 04 Terminal 1
-	AMD(   56,  104, AMED_EXACTPOS,                   DIR_SW), // 05 Terminal 2
-	AMD(   56,   88, AMED_EXACTPOS,                   DIR_SW), // 06 Terminal 3
-	AMD(   56,   72, AMED_EXACTPOS,                   DIR_SW), // 07 Terminal 4
-	AMD(   88,  120, AMED_EXACTPOS,                   DIR_N ), // 08 Terminal 5
-	AMD(   88,  104, AMED_EXACTPOS,                   DIR_NE), // 09 Terminal 6
-	AMD(   88,   88, AMED_EXACTPOS,                   DIR_NE), // 10 Terminal 7
-	AMD(   88,   72, AMED_EXACTPOS,                   DIR_NE), // 11 Terminal 8
-	AMD(   88,   56, AMED_EXACTPOS,                   DIR_SE), // 12 Helipad 1
-	AMD(   72,   56, AMED_EXACTPOS,                   DIR_NE), // 13 Helipad 2
-	AMD(   40,  136, 0,                               DIR_N ), // 14 Term group 2 enter 1 a
-	AMD(   56,  136, 0,                               DIR_N ), // 15 Term group 2 enter 1 b
-	AMD(   88,  136, 0,                               DIR_N ), // 16 Term group 2 enter 2 a
-	AMD(  104,  136, 0,                               DIR_N ), // 17 Term group 2 enter 2 b
-	AMD(  104,  120, 0,                               DIR_N ), // 18 Term group 2 - opp term 5
-	AMD(  104,  104, 0,                               DIR_N ), // 19 Term group 2 - opp term 6 & exit2
-	AMD(  104,   88, 0,                               DIR_N ), // 20 Term group 2 - opp term 7 & hangar area 2
-	AMD(  104,   72, 0,                               DIR_N ), // 21 Term group 2 - opp term 8
-	AMD(  104,   56, 0,                               DIR_N ), // 22 Taxi Term group 2 exit a
-	AMD(  104,   40, 0,                               DIR_N ), // 23 Taxi Term group 2 exit b
-	AMD(   56,   40, 0,                               DIR_N ), // 24 Term group 2 exit 2a
-	AMD(   40,   40, 0,                               DIR_N ), // 25 Term group 2 exit 2b
-	AMD(   40,  120, 0,                               DIR_N ), // 26 Term group 1 - opp term 1
-	AMD(   40,  104, 0,                               DIR_N ), // 27 Term group 1 - opp term 2 & hangar area 1
-	AMD(   40,   88, 0,                               DIR_N ), // 28 Term group 1 - opp term 3
-	AMD(   40,   72, 0,                               DIR_N ), // 29 Term group 1 - opp term 4
-	AMD(   18,   72, 0,                               DIR_NW), // 30 Outway 1
-	AMD(    8,   40, 0,                               DIR_NW), // 31 Airport OUTWAY
-	AMD(    8,   24, AMED_EXACTPOS,                   DIR_SW), // 32 Accelerate to end of runway
-	AMD(  119,   24, AMED_NOSPDCLAMP,                 DIR_N ), // 33 Release control of runway, for smoother movement
-	AMD(  117,   24, AMED_NOSPDCLAMP,                 DIR_N ), // 34 End of runway
-	AMD(  197,   24, AMED_NOSPDCLAMP | AMED_TAKEOFF,  DIR_N ), // 35 Take off
-	AMD(  254,   84, AMED_HOLD       | AMED_SLOWTURN, DIR_N ), // 36 Flying to landing position in air
-	AMD(  117,  168, AMED_HOLD       | AMED_LAND,     DIR_N ), // 37 Going down for land
-	AMD(    8,  168, AMED_NOSPDCLAMP | AMED_BRAKE,    DIR_N ), // 38 Just landed, brake until end of runway
-	AMD(    8,  168, 0,                               DIR_N ), // 39 Just landed, turn around and taxi
-	AMD(    8,  144, 0,                               DIR_NW), // 40 Taxi from runway
-	AMD(    8,  128, 0,                               DIR_NW), // 41 Taxi from runway
-	AMD(    8,  120, AMED_EXACTPOS,                   DIR_NW), // 42 Airport entrance
-	AMD(   56,  344, AMED_HOLD       | AMED_SLOWTURN, DIR_N ), // 43 Fly around waiting for a landing spot (north-east)
-	AMD( -200,   88, AMED_HOLD       | AMED_SLOWTURN, DIR_N ), // 44 Fly around waiting for a landing spot (north-west)
-	AMD(   56, -168, AMED_HOLD       | AMED_SLOWTURN, DIR_N ), // 45 Fly around waiting for a landing spot (south-west)
-	AMD(  312,   88, AMED_HOLD       | AMED_SLOWTURN, DIR_N ), // 46 Fly around waiting for a landing spot (south)
+	AMD(    6,   85, AMED_EXACTPOS,                   DIR_SE), // 00 In Hangar 1
+	AMD(  134,   70, AMED_EXACTPOS,                   DIR_SE), // 01 In Hangar 2
+	AMD(    6,  102, 0,                               DIR_N ), // 02 Taxi to right outside depot 1
+	AMD(  134,   86, 0,                               DIR_N ), // 03 Taxi to right outside depot 2
+	AMD(   54,  118, AMED_EXACTPOS,                   DIR_W ), // 04 Terminal 1
+	AMD(   54,  102, AMED_EXACTPOS,                   DIR_SW), // 05 Terminal 2
+	AMD(   54,   86, AMED_EXACTPOS,                   DIR_SW), // 06 Terminal 3
+	AMD(   54,   70, AMED_EXACTPOS,                   DIR_SW), // 07 Terminal 4
+	AMD(   86,  118, AMED_EXACTPOS,                   DIR_N ), // 08 Terminal 5
+	AMD(   86,  102, AMED_EXACTPOS,                   DIR_NE), // 09 Terminal 6
+	AMD(   86,   86, AMED_EXACTPOS,                   DIR_NE), // 10 Terminal 7
+	AMD(   86,   70, AMED_EXACTPOS,                   DIR_NE), // 11 Terminal 8
+	AMD(   86,   54, AMED_EXACTPOS,                   DIR_SE), // 12 Helipad 1
+	AMD(   70,   54, AMED_EXACTPOS,                   DIR_NE), // 13 Helipad 2
+	AMD(   38,  134, 0,                               DIR_N ), // 14 Term group 2 enter 1 a
+	AMD(   54,  134, 0,                               DIR_N ), // 15 Term group 2 enter 1 b
+	AMD(   86,  134, 0,                               DIR_N ), // 16 Term group 2 enter 2 a
+	AMD(  102,  134, 0,                               DIR_N ), // 17 Term group 2 enter 2 b
+	AMD(  102,  118, 0,                               DIR_N ), // 18 Term group 2 - opp term 5
+	AMD(  102,  102, 0,                               DIR_N ), // 19 Term group 2 - opp term 6 & exit2
+	AMD(  102,   86, 0,                               DIR_N ), // 20 Term group 2 - opp term 7 & hangar area 2
+	AMD(  102,   70, 0,                               DIR_N ), // 21 Term group 2 - opp term 8
+	AMD(  102,   54, 0,                               DIR_N ), // 22 Taxi Term group 2 exit a
+	AMD(  102,   38, 0,                               DIR_N ), // 23 Taxi Term group 2 exit b
+	AMD(   54,   38, 0,                               DIR_N ), // 24 Term group 2 exit 2a
+	AMD(   38,   38, 0,                               DIR_N ), // 25 Term group 2 exit 2b
+	AMD(   38,  118, 0,                               DIR_N ), // 26 Term group 1 - opp term 1
+	AMD(   38,  102, 0,                               DIR_N ), // 27 Term group 1 - opp term 2 & hangar area 1
+	AMD(   38,   86, 0,                               DIR_N ), // 28 Term group 1 - opp term 3
+	AMD(   38,   70, 0,                               DIR_N ), // 29 Term group 1 - opp term 4
+	AMD(   16,   70, 0,                               DIR_NW), // 30 Outway 1
+	AMD(    6,   38, 0,                               DIR_NW), // 31 Airport OUTWAY
+	AMD(    6,   22, AMED_EXACTPOS,                   DIR_SW), // 32 Accelerate to end of runway
+	AMD(  117,   22, AMED_NOSPDCLAMP,                 DIR_N ), // 33 Release control of runway, for smoother movement
+	AMD(  115,   22, AMED_NOSPDCLAMP,                 DIR_N ), // 34 End of runway
+	AMD(  195,   22, AMED_NOSPDCLAMP | AMED_TAKEOFF,  DIR_N ), // 35 Take off
+	AMD(  252,   82, AMED_HOLD       | AMED_SLOWTURN, DIR_N ), // 36 Flying to landing position in air
+	AMD(  115,  166, AMED_HOLD       | AMED_LAND,     DIR_N ), // 37 Going down for land
+	AMD(    6,  166, AMED_NOSPDCLAMP | AMED_BRAKE,    DIR_N ), // 38 Just landed, brake until end of runway
+	AMD(    6,  166, 0,                               DIR_N ), // 39 Just landed, turn around and taxi
+	AMD(    6,  142, 0,                               DIR_NW), // 40 Taxi from runway
+	AMD(    6,  126, 0,                               DIR_NW), // 41 Taxi from runway
+	AMD(    6,  118, AMED_EXACTPOS,                   DIR_NW), // 42 Airport entrance
+	AMD(   54,  342, AMED_HOLD       | AMED_SLOWTURN, DIR_N ), // 43 Fly around waiting for a landing spot ,north-east)
+	AMD( -202,   86, AMED_HOLD       | AMED_SLOWTURN, DIR_N ), // 44 Fly around waiting for a landing spot ,north-west)
+	AMD(   54, -170, AMED_HOLD       | AMED_SLOWTURN, DIR_N ), // 45 Fly around waiting for a landing spot ,south-west)
+	AMD(  310,   86, AMED_HOLD       | AMED_SLOWTURN, DIR_N ), // 46 Fly around waiting for a landing spot ,south)
 	/* Helicopter */
-	AMD(   96,   40, AMED_NOSPDCLAMP,                 DIR_N ), // 47 Bufferspace before helipad
-	AMD(   96,   40, AMED_NOSPDCLAMP,                 DIR_N ), // 48 Bufferspace before helipad
-	AMD(   82,   54, AMED_NOSPDCLAMP | AMED_SLOWTURN, DIR_N ), // 49 Get in position for Helipad1
-	AMD(   64,   56, AMED_NOSPDCLAMP | AMED_SLOWTURN, DIR_N ), // 50 Get in position for Helipad2
-	AMD(   81,   55, AMED_HELI_LOWER,                 DIR_N ), // 51 Land at Helipad1
-	AMD(   64,   56, AMED_HELI_LOWER,                 DIR_N ), // 52 Land at Helipad2
-	AMD(   80,   56, AMED_HELI_RAISE,                 DIR_N ), // 53 Takeoff Helipad1
-	AMD(   64,   56, AMED_HELI_RAISE,                 DIR_N ), // 54 Takeoff Helipad2
-	AMD(  136,   96, AMED_NOSPDCLAMP | AMED_SLOWTURN, DIR_N ), // 55 Go to position for Hangarentrance in air
-	AMD(  136,   96, AMED_HELI_LOWER,                 DIR_N ), // 56 Land in front of hangar2
-	AMD(  126,  104, 0,                               DIR_SE), // 57 Outway 2
-	AMD(  136,  136, 0,                               DIR_NE), // 58 Airport OUTWAY 2
-	AMD(  136,  152, AMED_EXACTPOS,                   DIR_NE), // 59 Accelerate to end of runway2
-	AMD(   16,  152, AMED_NOSPDCLAMP,                 DIR_N ), // 60 Release control of runway2, for smoother movement
-	AMD(   20,  152, AMED_NOSPDCLAMP,                 DIR_N ), // 61 End of runway2
-	AMD(  -56,  152, AMED_NOSPDCLAMP | AMED_TAKEOFF,  DIR_N ), // 62 Take off2
-	AMD(   24,    8, AMED_HOLD       | AMED_LAND,     DIR_N ), // 63 Going down for land2
-	AMD(  136,    8, AMED_NOSPDCLAMP | AMED_BRAKE,    DIR_N ), // 64 Just landed, brake until end of runway2in
-	AMD(  136,    8, 0,                               DIR_N ), // 65 Just landed, turn around and taxi
-	AMD(  136,   24, 0,                               DIR_SE), // 66 Taxi from runway 2in
-	AMD(  136,   40, 0,                               DIR_SE), // 67 Taxi from runway 2in
-	AMD(  136,   56, AMED_EXACTPOS,                   DIR_NE), // 68 Airport entrance2
-	AMD(  -56,    8, AMED_HOLD       | AMED_SLOWTURN, DIR_N ), // 69 Fly to landing position in air2
-	AMD(   88,   40, 0,                               DIR_N ), // 70 Taxi Term group 2 exit - opp heli1
-	AMD(   72,   40, 0,                               DIR_N ), // 71 Taxi Term group 2 exit - opp heli2
-	AMD(   88,   57, AMED_EXACTPOS,                   DIR_SE), // 72 pre-helitakeoff helipad 1
-	AMD(   71,   56, AMED_EXACTPOS,                   DIR_NE), // 73 pre-helitakeoff helipad 2
-	AMD(    8,  120, AMED_HELI_RAISE,                 DIR_N ), // 74 Helitakeoff outside depot 1
-	AMD(  136,  104, AMED_HELI_RAISE,                 DIR_N ), // 75 Helitakeoff outside depot 2
-	AMD(  197,  168, AMED_HOLD       | AMED_SLOWTURN, DIR_N ), // 76 Fly to landing position in air1
-	AMD(  393,   88, AMED_NOSPDCLAMP | AMED_SLOWTURN, DIR_N ), // 77 Initial DIAGDIR_NE approach
-	AMD(  312,    8, AMED_NOSPDCLAMP | AMED_SLOWTURN, DIR_N ), // 78 Initial DIAGDIR_SE approach
-	AMD( -278,   88, AMED_NOSPDCLAMP | AMED_SLOWTURN, DIR_N ), // 79 Initial DIAGDIR_SW approach
-	AMD( -200,  162, AMED_NOSPDCLAMP | AMED_SLOWTURN, DIR_N ), // 80 Initial DIAGDIR_NW approach
+	AMD(   94,   38, AMED_NOSPDCLAMP,                 DIR_N ), // 47 Bufferspace before helipad
+	AMD(   94,   38, AMED_NOSPDCLAMP,                 DIR_N ), // 48 Bufferspace before helipad
+	AMD(   80,   52, AMED_NOSPDCLAMP | AMED_SLOWTURN, DIR_N ), // 49 Get in position for Helipad1
+	AMD(   62,   54, AMED_NOSPDCLAMP | AMED_SLOWTURN, DIR_N ), // 50 Get in position for Helipad2
+	AMD(   79,   53, AMED_HELI_LOWER,                 DIR_N ), // 51 Land at Helipad1
+	AMD(   62,   54, AMED_HELI_LOWER,                 DIR_N ), // 52 Land at Helipad2
+	AMD(   78,   54, AMED_HELI_RAISE,                 DIR_N ), // 53 Takeoff Helipad1
+	AMD(   62,   54, AMED_HELI_RAISE,                 DIR_N ), // 54 Takeoff Helipad2
+	AMD(  134,   94, AMED_NOSPDCLAMP | AMED_SLOWTURN, DIR_N ), // 55 Go to position for Hangarentrance in air
+	AMD(  134,   94, AMED_HELI_LOWER,                 DIR_N ), // 56 Land in front of hangar2
+	AMD(  124,  102, 0,                               DIR_SE), // 57 Outway 2
+	AMD(  134,  134, 0,                               DIR_NE), // 58 Airport OUTWAY 2
+	AMD(  134,  150, AMED_EXACTPOS,                   DIR_NE), // 59 Accelerate to end of runway2
+	AMD(   14,  150, AMED_NOSPDCLAMP,                 DIR_N ), // 60 Release control of runway2, for smoother movement
+	AMD(   18,  150, AMED_NOSPDCLAMP,                 DIR_N ), // 61 End of runway2
+	AMD(  -58,  150, AMED_NOSPDCLAMP | AMED_TAKEOFF,  DIR_N ), // 62 Take off2
+	AMD(   22,    6, AMED_HOLD       | AMED_LAND,     DIR_N ), // 63 Going down for land2
+	AMD(  134,    6, AMED_NOSPDCLAMP | AMED_BRAKE,    DIR_N ), // 64 Just landed, brake until end of runway2in
+	AMD(  134,    6, 0,                               DIR_N ), // 65 Just landed, turn around and taxi
+	AMD(  134,   22, 0,                               DIR_SE), // 66 Taxi from runway 2in
+	AMD(  134,   38, 0,                               DIR_SE), // 67 Taxi from runway 2in
+	AMD(  134,   54, AMED_EXACTPOS,                   DIR_NE), // 68 Airport entrance2
+	AMD(  -58,    6, AMED_HOLD       | AMED_SLOWTURN, DIR_N ), // 69 Fly to landing position in air2
+	AMD(   86,   38, 0,                               DIR_N ), // 70 Taxi Term group 2 exit - opp heli1
+	AMD(   70,   38, 0,                               DIR_N ), // 71 Taxi Term group 2 exit - opp heli2
+	AMD(   86,   55, AMED_EXACTPOS,                   DIR_SE), // 72 pre-helitakeoff helipad 1
+	AMD(   69,   54, AMED_EXACTPOS,                   DIR_NE), // 73 pre-helitakeoff helipad 2
+	AMD(    6,  118, AMED_HELI_RAISE,                 DIR_N ), // 74 Helitakeoff outside depot 1
+	AMD(  134,  102, AMED_HELI_RAISE,                 DIR_N ), // 75 Helitakeoff outside depot 2
+	AMD(  195,  166, AMED_HOLD       | AMED_SLOWTURN, DIR_N ), // 76 Fly to landing position in air1
+	AMD(  388,  -22, AMED_NOSPDCLAMP | AMED_SLOWTURN, DIR_N ), // 77 Initial DIAGDIR_NE approach
+	AMD(  310, -132, AMED_NOSPDCLAMP | AMED_SLOWTURN, DIR_N ), // 78 Initial DIAGDIR_SE approach
+	AMD( -352,   86, AMED_NOSPDCLAMP | AMED_SLOWTURN, DIR_N ), // 79 Initial DIAGDIR_SW approach
+	AMD( -202,  268, AMED_NOSPDCLAMP | AMED_SLOWTURN, DIR_N ), // 80 Initial DIAGDIR_NW approach
 };
 
 
